tests/unit/archival-check.unit.ts

```diff
@@ -0,0 +1,176 @@
+import axios from 'axios'
+import MockAdapter from 'axios-mock-adapter'
+import RedisMock from 'ioredis-mock'
+import { expect, sinon } from '@loopback/testlab'
+import { Configuration, Session, RpcError } from '@pokt-network/pocket-js'
+
```

Comment:
Let's either remove this line or add it everywhere else, to keep consistency on imports :).